index.ts

```diff
@@ -21,29 +21,28 @@ export function* sinergia(
       const itemIterator: IterableIterator<any> = task(accumulator, item);
 
       yield new Promise(resolve => {
-        const step = (timestamp) => { // timestamp not used
+        const step = () => {
           const iteration = itemIterator.next();
 
-          if (!iteration.done) {
-            window.requestAnimationFrame(step);
-          }
-          else {
+          if (iteration.done) {
```

Comment:
What you said could just as well be reversed. It should feel more natural to read _if done, then resolve, else continue_ than _if **not** done, then continue, else resolve_.

It's not necessarily about understandability, but about avoiding double negations. 

Which sounds more natural?

Se non è buono lo mangi te, altrimenti lo mangio io.

Se è buono lo mangio, altrimenti lo mangi te.